generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -98,9 +101,26 @@ class WithFireSimDefaultFrequencyTweaks extends Config(
   new testchipip.WithAsynchronousSerialSlaveCrossing
 )
 
-// Tweaks that are generally applied to all firesim configs
+// Tweaks that are generally applied to all firesim configs setting a single clock domain at 1000 MHz
 class WithFireSimConfigTweaks extends Config(
-  new WithFireSimDefaultFrequencyTweaks ++
+  // 1 GHz matches the FASED default, using some other frequency will require
+  // runnings the FASED runtime configuration generator to generate faithful DDR3 timing values.
+  new chipyard.config.WithPeripheryBusFrequency(1000.0) ++
```

Comment:
Perhaps adding a comment saying that these buses (PBUS + MBUS) will be driven to 100 MHz (because of assignments in the Chisel, i'm guessing) unless overridden here, would be valuable.